pages/golfers/[userId].js

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react'
+import useScores from '../../lib/useScores'
+import Layout from '../../components/Layout'
+import { getUserId, getUsername } from '../../lib/userAuth'
+
+function GolfersDetails() {
+  const { scores } = useScores()
```

Comment:
Nice one, but this hook fetches everyone's scores. A more efficient approach would be to create a new hook (and a new API endpoint on the backend) that fetches only this user's scores. What do you think?

Reply:
Yes, it would make more sense to approach it that way.